.gitignore

```diff
@@ -25,4 +25,33 @@ Outline.apks
 universal.apk
 toc.pb
 coverage
-*.env
\ No newline at end of file
+*.env
+/build
+/bin
+intra/split/example/example
+
+# General
```

Comment:
what are all these files (Ln34 ~ Ln54)? It seems they are not generated by our code, are they Apple's specific stuffs?

Reply:
Can we try to remove these lines? I guess they will not be generated.